### PR TITLE
README: Move from `buildx` to `ko`

### PR DESCRIPTION
**Pull request by maelvls**
I've also clarified a lot of instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,7 +113,7 @@ configuration for the ECDSA signature is shown below in
 [print-your-cert-ca](#print-your-cert-ca). Also, I chose to have a very long
 expiry for both certificates since there is no security risk associated with
 leaking either of the private keys (since the private keys of both will be
-discarded on 21 May 2022 anyways).
+discarded on 23 March 2024 anyways).
```
Comment (SgtCoDFish) [inline README.md:116]: suggestion: I'm not sure if we should discard then!

We now have the ability to use the same root every kubecon and then swap out the intermediate. I think that's kinda cool!

File: README.md [hunk 2]
```diff
@@ -527,44 +581,38 @@ The first step is to create a cluster with a cert-manager issuer:
...
-    secretName: print-your-cert-ca
-EOF
+```
+
+Finally, set up the ClusterIssuer:
+
+```sh
+# From the Raspberry Pi:
+kubectl apply -f cluster_issuer.yaml
 ```
```
Comment (SgtCoDFish) [inline README.md:616]: suggestion: I think first we need to either generate or install the root?

**Review by SgtCoDFish — APPROVED** [review]
/lgtm
/approve
/hold

Adding a hold in case you want to change anything based on my comments, but it doesn't matter!

File: README.md (hunk 1 above)
Comment (maelvls) [reply README.md:116]: Agreed, discarding the private key would defeat the purpose of keeping the same root across kubecon

File: README.md (hunk 2 above)
Comment (maelvls) [reply README.md:616]: Ah, I think I wrote these instructions before the root was in its own YAML file. I'll fix that this morning

**Pull request merged**